packages/create-plugin/templates/common/playwright.config

```diff
@@ -43,7 +43,13 @@ export default defineConfig<PluginOptions>({
     // 2. Run tests in Google Chrome. Every test will start authenticated as admin user.
     {
       name: 'chromium',
-      use: { ...devices['Desktop Chrome'], storageState: 'playwright/.auth/admin.json' },
+      use: {
+        ...devices['Desktop Chrome'],
+        storageState: 'playwright/.auth/admin.json',
+        launchOptions: {
+          args: ['--disable-features=PlzDedicatedWorker'], // because https://github.com/microsoft/playwright/pull/34400
```

Comment:
Does the `auth` test above also need this?

Reply:
Good question I'm not sure, it seems to work without it because I think that test only uses the HTTP api 👍